embodichain/lab/sim/objects/articulation.py

```diff
@@ -1535,6 +1538,57 @@ def get_visual_material_inst(
                 result.append(mat_dict)
         return result
 
+    def set_collision_render_visibility(
+        self,
+        collision_visible: bool = True,
+        render_visible: bool = True,
+        link_names: Optional[List[str]] = None,
+        collision_visible_rgba: Optional[Sequence[float]] = None,
```

Comment:
Just use `rgba` and add description for param. Also methon it will be defined at the first time the function is called.